test/util/mp4_box_parsers_unit.js

```diff
@@ -149,4 +149,16 @@ describe('Mp4BoxParsers', () => {
     expect(defaultSampleDuration).toBe(expectedDefaultSampleDuration);
     expect(baseMediaDecodeTime).toBe(expectedBaseMediaDecodeTime);
   });
+
+  it('parses TKHD v1 box', () => {
+    const tkhdBox = new Uint8Array([
```

Comment:
I think this deserves some comments to explain the data in this Uint8Array.  For example, I had to dig to discover:

1. This is the box structure _after_ header, version and flags.
2. This is an incomplete box, stopping after the track ID, because the parser doesn't parse the other fields of the box.

These would be useful things to note in a comment.

Reply:
I have made the bare minimum Uint8Array to test the unboxing. Agree that some explanation is needed.